src/explorer/AppSettingsNodes.ts

```diff
@@ -138,39 +151,38 @@ export class AppSettingsNode extends NodeBase {
 
     private applySettings(): Promise<WebSiteModels.StringDictionary> {
         const webApps = this.WebSiteManagementClient.webApps;
-        const updateTask = this._isSlot ? webApps.updateApplicationSettingsSlot(this.site.resourceGroup, this._siteName, this._settings, this._slotName) :
+        return this._isSlot ? webApps.updateApplicationSettingsSlot(this.site.resourceGroup, this._siteName, this._settings, this._slotName) :
             webApps.updateApplicationSettings(this.site.resourceGroup, this._siteName, this._settings);
-        return updateTask;
-    }
-
-    openInPortal(): void {
-        const portalEndpoint = 'https://portal.azure.com';
-        const deepLink = `${portalEndpoint}/${this.subscription.tenantId}/#resource${this.site.id}/application`;
-        opn(deepLink);
     }
 }
 
 export class AppSettingNode extends NodeBase {
-    constructor(private key: string,
-        private value: string,
+    private key: string;
+    private value: string;
+
+    constructor(
+        key: string,
+        value: string,
         treeDataProvider: AppServiceDataProvider,
         parentNode: NodeBase) {
         super(`${key}=${value}`, treeDataProvider, parentNode);
+        this.key = key;
+        this.value = value;
     }
 
-    getTreeItem(): TreeItem {
+    public getTreeItem(): TreeItem {
         return {
             label: this.label,
             collapsibleState: TreeItemCollapsibleState.None,
-            contextValue: "applicationSettingItem",
+            contextValue: `applicationSettingItem`,
```

Comment:
single quotes (not template string)?

Reply:
Not a big deal, although I thought there was a rule against that.

Reply:
@nturinski I assume you changed this because of 'no-unexternalized-strings'? That rule should be turned off in this repo until we support localization